_data/simple-icons.json

```
@@ -3398,6 +3398,16 @@
             "hex": "BE2323",
             "source": "https://www.ebox.ca"
         },
+        {
+            "title": "Eclipse Adoptium",
+            "hex": "FF1464",
+            "source": "https://adoptium.net/",
```

```suggestion
            "source": "https://adoptium.net",
```

To fix redundant trailing slash linter error